### ISSUE TITLE
Bad file name breaks the build

### ISSUE THREAD

**Issue by innopal** [bencripps/react-redux-grid#40]
Hi Ben,
Here is a weird issue: There is apparently a trailing space in the name of one of your test files and that breaks our build process. Will you please remove that?
Here is the file:
"\node_modules\react-redux-grid\test\components\Grid.test.js “
Notice the space after .js!

Cheers

**Comment by bencripps** [comment]
This issue should be resolved with the latest commit. Published under v2.0.1

Let me know if you have any other issues!

**Comment by innopal** [comment]
Thanks Ben, 

I updated to 2.0.2 and getting this error now when I build my app.

  ```
TypeError: undefined is not a constructor (evaluating 'Array.from(classes)')
  at webpack:///~/react-redux-grid/dist/util/prefix.js:15:4
```

**Comment by bencripps** [comment]
Hmm, are you using webpack to build the grid? Or are you using the `dist` folder, which is already built code?

**Comment by bencripps** [comment]
In your project, please install the grid from this branch: `feature/bugfix-arrayfrom`.

See if this fixes your issue. If it does, I can merge it to master -- but i'm hesitant, because I think it points to a larger problem re: the build process.

**Comment by innopal** [comment]
I installed it from your bugfix branch but now I am getting this error:
```
ERROR in ./~/react-redux-grid/dist/index.js
Module build failed: ReferenceError: Unknown plugin "babel-plugin-transform-arra
y-from" specified in "C:\\Code\\4.7\\ecs-patient-portal\\node_modules\\react-red
ux-grid\\.babelrc" at 0, attempted to resolve relative to "C:\\Code\\4.7\\ecs-pa
tient-portal\\node_modules\\react-redux-grid"
    at C:\Code\4.7\ecs-patient-portal\node_modules\babel-core\lib\transformation
\file\options\option-manager.js:177:17
    at Array.map (native)
    at Function.normalisePlugins (C:\Code\4.7\ecs-patient-portal\node_modules\ba
bel-core\lib\transformation\file\options\option-manager.js:153:20)
    at OptionManager.mergeOptions (C:\Code\4.7\ecs-patient-portal\node_modules\b
abel-core\lib\transformation\file\options\option-manager.js:245:36)
    at OptionManager.init (C:\Code\4.7\ecs-patient-portal\node_modules\babel-cor
e\lib\transformation\file\options\option-manager.js:360:12)
    at File.initOptions (C:\Code\4.7\ecs-patient-portal\node_modules\babel-core\
lib\transformation\file\index.js:223:65)
    at new File (C:\Code\4.7\ecs-patient-portal\node_modules\babel-core\lib\tran
sformation\file\index.js:140:24)
    at Pipeline.transform (C:\Code\4.7\ecs-patient-portal\node_modules\babel-cor
e\lib\transformation\pipeline.js:46:16)
    at transpile (C:\Code\4.7\ecs-patient-portal\node_modules\babel-loader\index
.js:14:22)
    at Object.module.exports (C:\Code\4.7\ecs-patient-portal\node_modules\babel-
loader\index.js:88:12)
```

**Comment by bencripps** [comment]
You will need to re `npm install` since I've added new dependencies.

**Comment by innopal** [comment]
Couple of things:
Not building the grid is awesome! that cleaned up my webpack.config really nicely. Thanks for that.
Doing it the new way I'm not getting that previous error anymore but getting a new one:
  ```
Error: Cannot find module "array-from"
  at webpack:///~/react-redux-grid/dist/components/Grid.js:8:0
```

**Comment by bencripps** [comment]
Sorry about that, I added `array-from` as a dev dependency and it needed to be a dependency. Try reinstalling the grid from the feature branch.

**Comment by innopal** [comment]
Thanks Ben,

Worked like a charm! Any chance we get a release if not too much trouble?

**Comment by bencripps** [comment]
I'd like to test this a bit more with other environment variables. I will have a release soon. Probably published by tomorrow.

**Comment by bencripps** [comment]
Published under @2.0.3
